eosiojava/src/main/java/one/block/eosiojava/session/TransactionProcessor.java

```diff
@@ -586,13 +624,15 @@ private EosioTransactionSignatureRequest createSignatureRequest()
 
         // Cache the serialized version of transaction in the TransactionProcessor
         this.serializedTransaction = this.serializeTransaction();
+        this.serializedContextFreeData = this.serializeContextFreeData();
 
         EosioTransactionSignatureRequest eosioTransactionSignatureRequest = new EosioTransactionSignatureRequest(
-                this.serializedTransaction,
+                this.getSerializedTransaction(),
```

Comment:
serializedTransaction is assigned with value from ```this.serializeTransaction``` in line 627. Should we keep using that?

Reply:
I noticed `getSerializedTransaction()` wasn't really being used for anything besides testing, so I used it here. This could be reverted and the line a few below it for `this.getSerializedContextFreeData()` could just be `this.serializedContextFreeData`. I don't mind updating if we want.

Reply:
Updated.